.github/workflows/deploy_production.yml

```diff
@@ -0,0 +1,123 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/SchemaStore/schemastore/master/src/schemas/json/github-workflow.json
+---
+name: Deploy CloudFormation stacks for production environment and development environment mirror
+
+on:
+    push:
+        branches:
+            - main
+
+    workflow_dispatch:
+
+concurrency:
+    group: ${{ github.ref }}
+    cancel-in-progress: true
+
+permissions:
+    id-token: write
+    contents: read
+    pull-requests: write
+    deployments: write
+    repository-projects: write
+
+env:
+    HUSKY: 0
+    NODE_ENV: production
+    HOSTED_ZONE_ID: ${{ secrets.HOSTED_ZONE_ID }}
+    CERTIFICATE_ARN: ${{ secrets.CERTIFICATE_ARN }}
+    MONGODB_URI_PROD: ${{ secrets.MONGODB_URI_PROD }}
+
+    # Turborepo credentials.
+    TURBO_API: ${{ vars.TURBO_API }}
+    TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+    TURBO_TEAM: team_antalmanac
+
+    # AWS credentials.
+    AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+    AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+    AWS_DEFAULT_REGION: us-east-1
+
+jobs:
+    deploy_production_frontend:
+        name: Deploy production frontend to GitHub Pages
+
+        needs: setup_and_build
+
+        runs-on: ubuntu-latest
+
+        environment:
+            name: production-frontend
+            url: https://antalmanac.com
+
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v3
+
+            - name: Setup Node.js and pnpm
+              uses: ./.github/actions/setup-node-and-pnpm
+
+            - name: Install dependencies
+              run: pnpm install --frozen-lockfile
+
+            - name: Build frontend
+              run: pnpm --filter "antalmanac" build
+
+            - name: Upload frontend artifacts
+              uses: actions/upload-pages-artifact@v2
+              with:
+                  path: apps/antalmanac/build
+
+    deploy_production_backend:
+        name: Deploy production backend CloudFormation stack and environment
+
+        runs-on: ubuntu-latest
+
+        environment:
+            name: production-backend
+            url: https://api.antalmanac.com
+
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v3
+
+            - name: Setup Node.js and pnpm
+              uses: ./.github/actions/setup-node-and-pnpm
+
+            - name: Install dependencies
+              run: pnpm install --frozen-lockfile
+
+            - name: Build backend
+              run: pnpm --filter "antalmanac-backend" build
+
+            - name: Deploy production backend CloudFormation stack
+              run: pnpm --filter "antalmanac-cdk" production-backend deploy
+
+    # A shadow/mirror of the production backend used for development.
+    deploy_production_shadow_backend:
```

Comment:
Very nit: I think for the dev backend referring to it as dev backend makes more sense imo